docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -54,7 +55,7 @@
 
 # benchmark
 n_runs = 1 # optional, increase to create multiple runs and report mean + std
-batch_sizes = [256]
```

Comment:
When you want to commit the general benchmarks please make sure to not change the parameters so we don't have to run all benchmarks again.

Reply:
Yes, it's just a draft, I will change the parameters back :)